Jenkinsfile

```diff
@@ -141,7 +137,10 @@ pipeline {
     }
     stage('Build and Package on Ubuntu Jammy') {
       when {
-        branch 'release'
+        anyOf {
+          branch 'release'
+          changeRequest()
+        }
```

Comment:
Why do we need to test this on every commit after removing pyk?
Same for bellow.

Reply:
No, I enabled these tests for testing purposes. The corresponding commit (https://github.com/runtimeverification/k/pull/2858/commits/7a8e3c4a2ae83315b9083ced152e30e05e2100c2.) will be dropped before merge.